src/common/components/Footer/Footer.module.css

```
@@ -50,3 +50,76 @@
   width: 150%;
   cursor: pointer;
 }
+
+
+@media(max-width:768px){
+  .footer {
+    display: flex;
+    background-color: #474545;
+    width: 100%;
+    height: auto;
+    font-family: 'Quicksand', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Quicksand" (font-family-name-quotes)

(font-family-name-quotes)